Ollivanders/src/net/pottercraft/ollivanders2/player/O2Players.java

```diff
@@ -624,4 +629,421 @@ public void fixPlayerAnimagusColorVariant(@NotNull UUID pid, @NotNull String cor
       if (player != null)
          player.setAnimagusColor(correctedVariant);
    }
+
+   /**
+    * Commands related to players.
+    *
+    * @param sender the command sender
+    * @param args the args for the command
+    */
+   public boolean runSummary (@NotNull CommandSender sender, @NotNull String[] args)
```

Comment:
Nit: Space between function name and args (also the case in usageSummary, usageSummaryPlayer, usageSummaryAdmin)

Reply:
Fixed - it was like that in several places.